xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -1222,14 +1233,19 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Necessary force to open the door. +v1.1</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="SuitableForCycles" type="xsd:boolean" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether equipment is suitable for cycles.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
 			<xsd:element name="AudioPassthroughIndicator" type="xsd:boolean" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Whether there is an audio signal indicating passing through. +v1.1</xsd:documentation>
+					<xsd:documentation>Whether there is an audio signal indicating passing trhrough</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="SuitableForCycles" type="xsd:boolean" minOccurs="0">
+			<xsd:element name="OpeningNecessaryForce" type="NecessaryForceEnumeration" minOccurs="0">
```

Comment:
Can you check if this part is intended, SuitableForCycles is removed?

Reply:
It is still there. It was duplicated. See line 1236. It is just strange behaviour by the diff.